jrnl/cli.py

```diff
@@ -196,6 +196,8 @@ def run(manual_args=None):
                     print(f"[Could not read template at '{config['template']}']", file=sys.stderr)
                     sys.exit(1)
             raw = util.get_text_from_editor(config, template)
+            if not raw:
+                sys.exit()
```

Comment:
By the way, I would move this down to the bottom of the big if statement, so that it also exits when not using an editor and writing nothing.

Reply:
I'm not entirely sure where you mean.

Reply:
Where it currently just sets mode_compose to False in line 211